### PR TITLE
[Merged by Bors] - fix: initialize monitoring before proxy creation

### PR DESCRIPTION
**Pull request by morenol**

### CODE REVIEW

File: crates/fluvio-spu/src/start.rs [hunk 1]
```diff
@@ -46,12 +46,12 @@ pub fn main_loop(opt: SpuOpt) {
         let _public_shutdown = internal_server.unwrap().run();
         let _private_shutdown = public_server.unwrap().run();
 
+        init_monitoring(ctx);
+
         if let Some(tls_config) = tls_acceptor_option {
             proxy::start_proxy(spu_config, tls_config).await;
```
Comment (morenol) [inline crates/fluvio-spu/src/start.rs:52]: this never finished

**Review by sehz — APPROVED** [review]
LGTM

**Pull request closed without merging**